solanaj/src/main/java/org/p2p/solanaj/kits/transaction/ConfirmedTransactionParser.kt

```diff
@@ -1,345 +0,0 @@
-package org.p2p.solanaj.kits.transaction
-
-import org.p2p.solanaj.kits.transaction.ConfirmedTransactionParsed.InstructionParsed
-import org.p2p.solanaj.programs.SerumSwapProgram
-import org.p2p.solanaj.programs.SerumSwapProgram.serumSwapPID
-import org.p2p.solanaj.programs.SystemProgram.SPL_TOKEN_PROGRAM_ID
-import java.math.BigInteger
-
-// todo: Parser should be refactored and optimized
-// Get some parsing info from here
-// https://github.com/p2p-org/solana-swift/blob/main/Sources/SolanaSwift/Helpers/TransactionParser.swift#L74-L86
-object ConfirmedTransactionParser {
-
-    private var details = mutableListOf<TransactionDetails>()
```

Comment:
refactored into `ConfirmedTransactionRootParser` + removed mutability
`checkOrcaSwapDetails` and `parseSerumSwapTransaction` moved to a separate files like on iOS and Web